.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
     - chmod 777 ~/onFailure.sh
     - chmod 777 ~/onSuccess.sh
 install:
-    - mvn dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+    - mvn clean install dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
Agree with you. So, maybe we could change from install to verify goal, so maven would not put it into the repsoitory. The install goal we can use in the line 16 , as it will include the tests. Or how about we use both 'install' goal (from maven) on the install and script phase (from travis) and on failure, we wiped the version installed and move back a previous jar in the local repository  ?